src/apphosting/secrets/dialogs.ts

```diff
@@ -143,37 +142,38 @@ export async function selectBackendServiceAccounts(
     if (grant) {
       return toMulti(serviceAccountsForBackend(projectNumber, listBackends.backends[0]));
     }
-    utils.logLabeledBullet("apphosting", GRANT_ACCESS_IN_FUTURE);
+    utils.logBullet(GRANT_ACCESS_IN_FUTURE);
     return { buildServiceAccounts: [], runServiceAccounts: [] };
   }
 
   const metadata: BackendMetadata[] = toMetadata(projectNumber, listBackends.backends);
 
   if (metadata.every(matchesServiceAccounts(metadata[0]))) {
-    utils.logLabeledBullet(
-      "apphosting",
-      "To use this secret, your backend's service account must have secret accessor permission. All of your backends use " +
-        (sameServiceAccount(metadata[0]) ? "service account " : "service accounts ") +
+    utils.logBullet(
+      "To use this secret, your backend's service account must have secret accessor permission.",
```

Comment:
This feels awkward to me for some reason. "To use this secret, your backend's service account must be granted access."?